src/panels/config/devices/ha-config-device-page.ts

```diff
@@ -1119,17 +1119,32 @@ export class HaConfigDevicePage extends LitElement {
       const matter = await import(
         "./device-detail/integration-elements/matter/device-actions"
       );
-      const actions = await matter.getMatterDeviceActions(
+      const defaultActions = matter.getMatterDeviceDefaultActions(
         this,
         this.hass,
         device
       );
-      deviceActions.push(...actions);
+      deviceActions.push(...defaultActions);
+      this._loadMatterDeviceActions(matter, device);
```

Comment:
Maybe cleaner to just keep it all in here instead of in a separate function?
```suggestion
matter.getMatterDeviceActions(
      this,
      this.hass,
      device
    ).then(actions => { this._deviceActions = [...actions, ...(this._deviceActions || [])];});
```